### ISSUE TITLE
improve updates from dependencies 

### ISSUE THREAD

**Issue by typemytype** [roboDocs/docs#2]
and maybe remove all the submodules in the repo... and switch to some sort of requirement.txt based thingy to extract docs from

see https://requires.io

@anthrotype : I guess, this seems like a good tool for this project

**Comment by anthrotype** [comment]
Yes, that's sounds like a good idea. The only problem is that we first need to publish each project to PyPI for requires.io to be able to detect version updates.
I'm working towards that, but it may take some time before all projects will be published, and pip-install-able.

**Comment by typemytype** [comment]
mmm, oke, I see
